### DISCUSSION TITLE
PSA: If your WordPress Quick App gives you errors or is always an old version...

### DISCUSSION THREAD

**Discussion by shoelaced** [leokhoa/laragon#564] [General]
After DAYS of trying to figure out why I was getting a database error on a brand new installation instead of the usual WordPress installer wizard, I discovered that `wordpress.latest.tar.gz` has been cached on my machine since 2019. 2019! No wonder it always installed a four-year-old version of WordPress -- I thought Laragon just hadn't been updated all this time! What on earth... 

So PSA for anyone else: Go delete your cached version of WordPress in `C:\laragon\tmp\cached` and then run the Quick App installer again. You're welcome.

And feedback for the Laragon team: Please add a setting to set some sort of cache expiration for this. Many thanks.

**Comment by leokhoa** [comment]
@shoelaced : Thanks for your information.
Caching is default and you can disable the cache. As network is fast nowadays, the next version of Laragon will disable the cache.

**Reply by shikkaba** [reply]
@leokhoa so happy to still see you're around. Thank you for this amazing software. Can't wait for the next version!
